stdlib/LinearAlgebra/src/uniformscaling.jl

```diff
@@ -194,11 +194,11 @@ end
 function isapprox(J::UniformScaling, A::AbstractMatrix;
                   atol::Real = 0,
                   rtol::Real = Base.rtoldefault(promote_leaf_eltypes(A), eltype(J), atol),
-                  nans::Bool = false, norm::Function = vecnorm)
+                  nans::Bool = false, norm::Function = norm)
```

Comment:
I know the ambiguity was resolved, but we might want to use `normfun` or similar as the keyword argument.